src/components/MainPage.vue

```diff
@@ -32,72 +31,85 @@
     >
       <Importers />
     </v-navigation-drawer>
-    <v-content>
-      <DataTable />
-    </v-content>
     <v-navigation-drawer
       v-model="drawerRight"
       app
       right
     >
       <Exporters />
     </v-navigation-drawer>
+    <v-footer
+      app
+      color="primary"
+    >
+      <profile-chip
+        v-for="profile in profiles"
+        :key="profile.email"
+        :profile="profile"
+      />
+      <v-spacer />
+      <v-btn
+        text
+        small
+        @click="() => openExternal('https://github.com/baruchiro/israeli-bank-scrapers-desktop')"
+      >
+        <span class="mr-2">Open in Github</span>
+        <v-icon>mdi-open-in-new</v-icon>
+      </v-btn>
+    </v-footer>
+    <!-- The content come after all the 'app' components,
+    to render when the $vuetify.application is ready -->
+    <v-content>
+      <DataTable />
+    </v-content>
   </v-app>
 </template>
 
 <script>
+import { shell } from 'electron';
 import Importers from '@/components/MainPage/Importers';
 import DataTable from '@/components/MainPage/DataTable';
 import Exporters from '@/components/MainPage/Exporters';
+import ProfileChip from '@/components/MainPage/ProfileChip';
 
 export default {
   name: 'MainPage',
-  components: { Importers, DataTable, Exporters },
+  components: {
+    Importers, DataTable, Exporters, ProfileChip,
+  },
   data() {
     return {
       drawerLeft: null,
       drawerRight: null,
+      profiles: [
+        {
+          avatar: 'https://avatars3.githubusercontent.com/u/17686879',
+          name: 'Baruch Odem',
+          email: 'baruchiro@gmail.com',
+          networks: [
+            'https://github.com/baruchiro',
+            'https://www.linkedin.com/in/baruch-rothkoff/',
+          ],
+        },
+        {
+          avatar: 'https://media-exp1.licdn.com/dms/image/C5603AQHBVlsYLgmvNw/profile-displayphoto-shrink_800_800/0?e=1587600000&v=beta&t=iRvw54ZS2K6k3PKi4jgtpe-noj9XTzyUf3IMJvaWShQ',
+          name: 'Ariel Gordon',
+          email: 'arielgordon123@gmail.com',
+          networks: [
+            'https://github.com/Arielgordon123',
+            'https://www.linkedin.com/in/ariel-gordon-462b14156/',
+          ],
```

Comment:
@Arielgordon123 Your details. Maybe it's time to set/replace a profile picture.
Do you think there is more networks we can add?